content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If co-workers and customers depend on Kubernetes being available to run critical
+workloads, it is your job to make sure that the platform is set up and
+maintained to meet their demands.
+
+<!-- body -->
+
+## Production issues
+
+A personal Kubernetes learning or development environment is not meant to be
+used in production. A production environment may require secure access by many
+users, consistent availability, and the resources to adapt to changing demands.
+
+As you decide where you want your production Kubernetes environment to live
+(on premises or in a cloud) and the amount of management you want to take
+on or hand to others, consider how your requirements for a Kubernetes cluster
+are colored by the following issues:
+
+- *Availability*: A single-machine Kubernetes [learning environment](/docs/setup/#learning-environment)
+has a single point of failure. Creating a highly available cluster means considering:
+  - Separating the control plane from the worker nodes.
+  - Duplicating the control plane by having multiple instances of each control plane service available, having those services run on multiple machines, or load balancing traffic to the cluster’s {{< glossary_tooltip term_id="kube-apiserver" text="API server" >}}.
+  - Having enough worker nodes available, or the availability to quickly come on line, as changing workloads warrant it.
```

Comment:
```suggestion
  - Creating a [high availability etcd cluster](/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) with at least three members. etcd clusters should have an [odd number of members.](https://etcd.io/docs/next/faq/#why-an-odd-number-of-cluster-members)
  - Having enough worker nodes available, or the availability to quickly come on line, as changing workloads warrant it.
```
Control plane nodes and etcd can be separated or co-located, [reference link](https://deploy-preview-27466--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/high-availability/). An HA cluster should have an odd number of etcd members whether it's stacked with the control plane components or external.

Reply:
Good information. I go into detail about etcd later, so I move the two links you suggest and surrounding wording there.